test/cpp/jit/test_lite_interpreter.cpp

```diff
@@ -596,6 +597,14 @@ TEST(LiteInterpreterTest, GetRuntimeByteCodeVersion) {
       caffe2::serialize::kMaxSupportedBytecodeVersion);
 }
 
+/**
```

Comment:
If it's the same reason for all tests, maybe just move this comment to the top of the doc?